src/pages/en/guides/typescript.md

```diff
@@ -112,13 +114,25 @@ const { greeting = 'Hello', name } = Astro.props;
 
 - If your component must be passed children to its default slot, you can enforce this by using `type Props = { children: any; };`.
 
-### Built-in attribute types
+## Type Utilities
 
-Astro provides JSX type definitions to check that your markup is using valid HTML attributes. You can use these types to help build component props. For example, if you were building a `<Link>` component, you could do the following to mirror the default HTML attributes in your component’s prop types.
+<Since v="1.6.0" />
 
-```astro title="src/components/Link.astro" ins={2}
+Astro comes with some built-in utility types for common prop type patterns. These are available under the `astro/types` entrypoint.
+
+### Built-in HTML attributes
+
+Astro provides the `HTMLAttributes` type to check that your markup is using valid HTML attributes. You can use these types to help build component props. For example, if you were building a `<Link>` component, you could do the following to mirror the default HTML attributes for `<a>` tags in your component’s prop types.
```

Comment:
```suggestion
Astro provides the `HTMLAttributes` type to check that your markup is using valid HTML attributes. You can use these types to help build component props.

For example, if you were building a `<Link>` component, you could do the following to mirror the default HTML attributes for `<a>` tags in your component’s prop types.
```

Reply:
super-nit: I'd consider breaking up the explanation from the "for example" since they each are about the same length.

Otherwise, LGTM! I'll consider this ready to `merge-on-release`! 🥳